7.7.поиск_ошибок_и_ревью_кода/7.7.3.py

```diff
@@ -0,0 +1,12 @@
+count = 0
+p = 1  
+for i in range(1, 11):  
```

Comment:
1. Так как индекс у нас не используется, называй переменные цикла `_`.
2. Лучше использовать `range(10)`, он генерирует равнозначное кол-во элементов что и range(1, 11), но так понятней.

```suggestion
for _ in range(10):  
```